plugins/module_utils/job.py

```diff
@@ -182,8 +209,31 @@ def _zget_job_status(job_id="*", owner="*", job_name="*", dd_name=None):
     else:
         job_query = job_id
 
+    # jls output: owner=job[0], name=job[1], id=job[2], status=job[3], rc=job[4]
+    # e.g.: OMVSADM  HELLO    JOB00126 JCLERR   ?
     # entries = listing(job_query, owner)   1.2.0 has owner paramn, 1.1 does not
-    entries = listing(job_query)
+
+    # Disabled for now to use work around below resulting from an index bound
+    # excpetion using zoau  dataset.listing()
+    #     entries = listing(job_query)
+
+    entries = []
+    response = _listing(job_query)
+
+    for unparsed_job in list(filter(None, response.stdout_response.split("\n"))):
+        job = list(filter(None, unparsed_job.rstrip("\n").split()))
+
+        count = 0
+        while len(job) != 5 and count < 9:
+            job = list(filter(None, _listing(job_query).stdout_response.rstrip("\n").split()))
+            time.sleep(1)
+            count += 1
+
+        if count >= 9:
```

Comment:
Even though this is my work around and what we have always seen is the owner is the one that has been coming back None, we could have done some added interrogation because we have the job_id and could see if it matches to job[2] and then  know if the left index or right index are empty had have a more refined guess at which to put a `?' into. The way it is now if owner is really not empty we are shifting the results over by 1 , this is not mean to  ship and we have a JIRA for it #375